CHANGELOG.md

```diff
@@ -1964,7 +1968,9 @@ This section is only relevant to contributors.
 
 **Historical Note 2** We removed the older changelog, you can look at the git history to get it. (version numbers made hallmark crazy)
 
-[unreleased]: https://github.com/deltachat/deltachat-desktop/compare/1.36.1...HEAD
+[unreleased]: https://github.com/deltachat/deltachat-desktop/compare/1.36.2...HEAD
```

Comment:
neither the new nor the new links work, there seems to be a `v` missing

```suggestion
[unreleased]: https://github.com/deltachat/deltachat-desktop/compare/v1.36.2...HEAD
```

not sure it maintaining these links are worth the effort or if they're really noticed or used by someone, i would also be fine with just pointing to a the common `compare` functionality, but that is another discussion, it is also okay as it is :)